build/windows/octopus.wxs

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<?ifndef ProductVersion?>
+<?error ProductVersion property not defined?>
+<?endif?>
+
+<!-- Define a unique UpgradeCode per platform -->
+<?if $(var.Platform) = "x64"?>
+<?define InstallerVersion = "200"?>
+<?define UpgradeCode = "738e840f-6305-4a12-8136-8aad988daf43"?>

```

Comment:
Just sanity checking; I assume you replaced these with random GUID's? They don't appear to match the github CLI wxs file anyway

Reply:
Yes, we replaced these GUIDs